jlm/llvm/opt/alias-analyses/AgnosticModRefSummarizer.hpp

```diff
@@ -76,6 +73,49 @@ class AgnosticModRefSummarizer final : public ModRefSummarizer
    */
   static std::unique_ptr<ModRefSummary>
   Create(const rvsdg::RvsdgModule & rvsdgModule, const PointsToGraph & pointsToGraph);
+
+private:
+  /**
+   * Creates a set containing all MemoryNodes in the given \p pointsToGraph
+   */
+  [[nodiscard]] static util::HashSet<const PointsToGraph::MemoryNode *>
+  GetAllMemoryNodes(const PointsToGraph & pointsToGraph);
+
+  /**
+   * Helper for adding all MemoryNodes the given \p output may target to a ModRefSet
```

Comment:
memory nodes....okay I get tired of this....

If you mean to actually reference the proper MemoryNode class, then please use \ref, otherwise there is no guarantee that doxygen will pick it up